### ISSUE TITLE
Download Link?

### ISSUE THREAD

**Issue by balivo** [MicrosoftDocs/xamarin-docs#1566]
Direct download link from Microsoft OpenJDK been removed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 97e1cd96-e7a2-ec22-8d79-ac949e890ccb
* Version Independent ID: 922a3fdf-7bab-7d9d-5518-904db9e75ee3
* Content: [Microsoft's Mobile OpenJDK Distribution - Xamarin](https://docs.microsoft.com/en-us/xamarin/android/get-started/installation/openjdk)
* Content Source: [docs/android/get-started/installation/openjdk.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/android/get-started/installation/openjdk.md)
* Product: **xamarin**
* Technology: **xamarin-android**
* GitHub Login: @vyedin
* Microsoft Alias: **vyedin**

**Comment by De3d** [comment]
Yeah... wonderful design of the document, too. Legalistic in nature. Love thine users is a sin, for them, it seems.

**Comment by jnooraga** [comment]
The button in VS says "Download Now", yet there is nowhere to download it on this page.

**Comment by zanval** [comment]
+1, VS for Mac redirects to this page, restarts the IDE, but I have no `Xamarin` folder at `$HOME/Library/Developer`

**Comment by lumiere-rayz** [comment]
same thing here @zanval,  I have no Xamarin folder at $HOME/Library/Developer.

**Comment by moljac** [comment]
I did minimal tests on Windows with Zulu and Adopt Open JDK. You need to use Java 8, because MSBuild tasks will not let you use newer versions.

*   https://www.azul.com/downloads/zulu-community/?package=jdk

*   https://adoptopenjdk.net/

*   https://adoptopenjdk.net/releases.html

I was able to launch AndroidX migrated apps in the emulator, but that is about it.

**Comment by davidbritch** [comment]
Thanks for submitting an issue and we apologize for the delayed response. The requested updates haven't been made since the creation of the issue, and the timeline for resolution may vary based on resourcing, so the issue is being moved to an internal repo to better track it.
